tests/test_classify.py

```diff
@@ -9,7 +9,7 @@
 class TestClassify(unittest.TestCase):
 
     def test_success(self):
-        prediction = co.classify(model='medium',
+        prediction = co.classify(model='small',
```

Comment:
We have rules for rerouting of requests. We should understand if and how the re-routing broke. 

Approving but these changes shouldn't technically be needed

Reply:
PS: how come we do not need to do this for embed?

Reply:
embed still has a small model that we're using to test